sw/device/silicon_creator/lib/drivers/otbn.c

```diff
@@ -59,6 +59,23 @@ static rom_error_t check_offset_len(uint32_t offset_bytes, size_t num_words,
   return kErrorOk;
 }
 
+rom_error_t otbn_busy_wait_for_done(void) {
+  uint32_t status = UINT32_MAX;
```

Comment:
Can you give this a try as we discussed offline?

```suggestion
  uint32_t status = launder32(UINT32_MAX);
```